src/Ryujinx.Headless.SDL2/WindowBase.cs

```diff
@@ -55,7 +56,14 @@ public static void QueueMainThreadAction(Action action)
         public IHostUiTheme HostUiTheme { get; }
         public int Width { get; private set; }
         public int Height { get; private set; }
+        public int DisplayId { get; set; }
         public bool IsFullscreen { get; set; }
+        public bool IsExclusiveFullscreen { get; set; }
+        public int ExclusiveFullscreenWidth { get; set; }
+        public int ExclusiveFullscreenHeight { get; set; }
+        public Ryujinx.Common.Configuration.AntiAliasing AntiAliasing { get; set; }
+        public Ryujinx.Common.Configuration.ScalingFilter ScalingFilter { get; set; }
```

Comment:
Can't use do a ``using Ryujinx.Common.Configuration;``  to import that namespace?

If you do then you can change this to:

```suggestion
        public AntiAliasing AntiAliasing { get; set; }
        public ScalingFilter ScalingFilter { get; set; }
```

Reply:
You can do:

`using AntiAliasing = Ryujinx.Common.Configuration.AntiAliasing;`

And same for `ScalingFilter`